cEpiabm/test/dataclasses/test_person.cpp

```diff
@@ -40,3 +46,29 @@ TEST_CASE("dataclasses/person: test setHousehold", "[Person]")
         static_cast<size_t>((std::rand()%500) * 2 + 1)) == false);
```

Comment:
Does that rely on it producing the same random number as the previous call?

Reply:
No, checking setHousehold returns false if the person is already linked to a household. (easier not to allow changing household)